### DISCUSSION TITLE
Problems while Importing/Updating Tables from Dataverse SQL-Database

### DISCUSSION THREAD

**Discussion by fischerf11** [TabularEditor/TabularEditor3#1149] [🙋♀️TE3 Q&A] [answered]
Hi everybody, 

I am new to tabular editor since today :-) and I really like the software. I am not totally sure, if I am in the right section, to ask for help. I am experiencing a problem while working with the Dataverse SQL-Database. When I am importing a table from there and try to use the "update table schema"-functionality, I am experiencing this error:

![image](https://user-images.githubusercontent.com/114239722/191977335-dede08e8-c0cc-4961-86d9-36c6f61107c9.png)

I am able to import the table, change e.g. column names and push pack to the power BI workspace. I am not able to get back already deleted columns, update the table from the source, ...

These are the connection parameters I am using:

![image](https://user-images.githubusercontent.com/114239722/191976534-46b3ece6-1107-4555-949f-1b6cb2ae9713.png)

Can anybody help me?

Best regards and thanks in advance!!
Fabian

**Comment by otykier** [comment, marked as answer]
I am not familiar with dataverse. I wonder why it’s not able to serve column metadata from the INFORMATION_SCHEMA.COLUMNS view. If you connect to dataverse with for example SSMS, are you able to query this view?

**Comment by fischerf11** [comment]
Hi Daniel. 
I think this does not work as well, if I did the test correct:
![image](https://user-images.githubusercontent.com/114239722/192047319-5d04f046-acad-4399-868e-6614da14bb67.png)
